apps/weight-loss-consultant-users-mgnt-api/src/app/services/sorting-filtering.service.ts

```diff
@@ -105,14 +107,16 @@ export class SortingAndFilteringService {
         break;
       }
       default: {
-        const result = await this.customerRepository.createQueryBuilder()
-          .where("gender = :gender", {gender: genderFilter === "Male" ? "Male" : "Female"})
+        const result = await this.customerRepository.createQueryBuilder("customer")
+          .where("gender = :gender", {gender: genderFilter === "1" ? Gender.MALE : Gender.FEMALE})
+          .leftJoinAndSelect("customer.campaigns", "campaign")
           .orderBy(sortBy, order === "ASC" ? "ASC" : "DESC")
           .offset(skipped)
           .limit(limit)
           .getMany()
-        const totalCount = await this.customerRepository.createQueryBuilder()
-          .where("gender = :gender", {gender: genderFilter === "Male" ? "Male" : "Female"})
+        const totalCount = await this.customerRepository.createQueryBuilder("customer")
+          .where("customer.gender = :gender", {gender: genderFilter === "1" ? Gender.MALE : Gender.FEMALE})
```

Comment:
What is "1" here means? Should separate to another variable or add comment.

Reply:
ok I will fix it